stable/sonatype-nexus/README.md

```diff
@@ -144,6 +148,13 @@ The following table lists the configurable parameters of the Nexus chart and the
 | `service.annotations`                       | Service annotations              | `nil`                                      |
 | `service.targetPort`                        | Service port                     | `nil`                                      |
 | `service.port`                              | Port for exposing service        | `nil`                                      |
+| `route.enabled`         | Set to true to create route for additional service | false |
+| `route.name`            | Name of route                                      | false |
+| `route.portName`        | Target port name of service                        | false |
+| `route.labels`          | Labels to be added to route                        | false |
+| `route.annotations`     | Annotations to be added to route                   | false |
+| `route.path`            | Host name of Route                                 | false |
+
 
 If `nexusProxy.env.cloudIamAuthEnabled` is set to `true` the following variables need to be configured
 
```

Comment:
Maybe add a section on what an example set of overrides for openshift look like.